docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,528 @@
+  The `noti_ex_led_info_set_on_period()`, `noti_ex_led_info_set_off_period()` function sets the time period when the LED is switched on and off.
+
+  ```
+  noti_ex_color_create(&color, 0, 0, 0, 0);
+
+  noti_ex_led_info_create(&led_info, color);
+  noti_ex_led_info_set_on_period(led_info, 5);
+  noti_ex_led_info_set_off_period(led_info, 5);
+  ```
+
+- channel:
+
+  Each notification can be displayed by channel.
+  ```
+  ret = noti_ex_item_set_channel(item_handle, "channel");
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- receiver group:
+
+  If you specify a receiver group that you want to receive, all viewers in that group receive a notification.
+  ```
+  ret = noti_ex_item_add_receiver(item_handle, NOTI_EX_RECEIVER_GROUP_INDICATOR);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+<a name="receiver"></a>
+  There are 5 viewer types:
+  `NOTI_EX_RECEIVER_GROUP_PANEL`
+  `NOTI_EX_RECEIVER_GROUP_TICKER`
+  `NOTI_EX_RECEIVER_GROUP_LOCKSCREEN`
+  `NOTI_EX_RECEIVER_GROUP_INDICATOR`
+  `NOTI_EX_RECEIVER_GROUP_POPUP`
+
+- visible:
+
+  Determines whether to display the item.
+  ```
+  ret = noti_ex_item_set_visible(item_handle, true);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- action:
+
+  app control action and visibility action can be set. Can be set for each item.
+  ```
+  noti_ex_item_h item_handle = NULL;
+  noti_ex_action_h action = NULL;
+
+  noti_ex_item_button_create(&item_handle, "button_id", "button_title");
+
+  app_control_create(&app_control);
+  app_control_set_app_id(app_control, "appid");
+  noti_ex_action_app_control_create(&action, app_control, NULL);
+
+  ret = noti_ex_item_set_action(item_handle, action);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+
+  app_control_destroy(app_control);
+  ```
+
+- main type:
+
+  You can specify main item among several items. If there are multiple text items and main item is specified, the viewer can display the items in particular.
+
+  The main type contains only title, contents, icon, and button.
+  ```
+  noti_ex_item_h text_item = NULL;
+  noti_ex_item_h text_item2 = NULL;
+  noti_ex_item_h group_item = NULL;
+
+  noti_ex_item_group_create(&group_item, "group_id");
+  noti_ex_item_text_create(&text_item, "text_id", "title", "hyperlink");
+  noti_ex_item_text_create(&text_item2, "text_id2", "contents", "hyperlink");
+  noti_ex_item_group_add_child(group_item, text_item);
+  noti_ex_item_group_add_child(group_item, text_item2);
+
+  ret = noti_ex_item_set_main_type(group_item, "text_id", type);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+<a name="expanded"></a>
+## Creating an Expanded Notification
+
+You can provide a more complex form of notification. You can also specify the display order of each item. Viewers can express them in the order of adding children to the group.
```

Comment:
" Viewers can express them in the order of adding children to the group." 
Are we saying:
The display order of each item depends on the order in which the child is added to the group.

Please verify and share your opinion so that I can rephrase it.